_aggregations/metric/weighted-avg.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Weighted average
+parent: Metric aggregations
+grand_parent: Aggregations
+nav_order: 150
+---
+
+# Weighted average
+
+The `weighted_avg` aggregation calculates the weighted average of numeric values across documents. This is useful when you want to calculate an average, but weight some data points more heavily than others.
+
+## Weighted average calculation
+
+The weighted average is calculated as `(sum of value * weight) / (sum of weights)`.
+
+## Parameters
+
+When using the `weighted_avg` aggregation you must define the following parameters:
+
+- `value`: The field or script to get the numeric values to average
```

Comment:
I made some assumptions here re: meaning. Please confirm that my changes reflect the intended meaning.